hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -885,4 +886,27 @@ public void close() {
   public long getDiffCleanupServiceInterval() {
     return diffCleanupServiceInterval;
   }
+
+  // Wait for double buffer to be flushed and confirm that the
+  // snapshot directory exists.
+  public void waitForSnapshotFlush(String volumeName, String bucketName,
```

Comment:
Looks like this is only used in tests. We can add an annotation for that.

```suggestion
  @VisibleForTesting
  public void waitForSnapshotFlush(String volumeName, String bucketName,
```